website/astro.config.ts

```diff
@@ -8,6 +9,11 @@ import rehypeAutolinkHeadings from "rehype-autolink-headings";
 import rehypeSlug from "rehype-slug";
 import remarkToc from "remark-toc";
 
+import { fileURLToPath } from "url";
```

Comment:
Nit: Would it be possible to use the `node:` prefix when using the node.js modules?

Reply:
Yeah , Exactly we can use the `node:` prefix when using the nodejs modules, like `import { fileURLToPath } from "node:url";` but @anonrig just suggest a better way where we dont require any nodejs module import , we can use the Global `new URL` object to resolve this.